crates/next-core/js/src/entry/config/next.js

```diff
@@ -13,6 +14,12 @@ const loadNextConfig = async (silent) => {
 
   nextConfig.generateBuildId = await nextConfig.generateBuildId?.();
 
+  const customRoutes = await loadCustomRoutes(nextConfig);
```

Comment:
We should be creating this on the Next.js side so that we don't have to call `loadCustomRoutes` twice x-ref: https://github.com/vercel/next.js/blob/7654d7bc2d95c8b3dc18cf5e94695375c733e74c/packages/next/src/server/lib/route-resolver.ts#L28

Reply:
We load the whole config a second time to call the route resolver right now unfortunately

Reply:
We load the whole config a second time to call the route resolver right now unfortunately

Reply:
@ijjk is there any chance we could get something implemented on the next.js side that would allow the whole config to be serialized to JSON (after loading custom routes probably)?